internal/lacework/server.go

```diff
@@ -57,6 +57,15 @@ func MockServer() *Mock {
 	}
 }
 
+func MockUnstartedServer() *Mock {
+	mux := http.NewServeMux()
+	return &Mock{
+		Mux:        mux,
+		Server:     httptest.NewUnstartedServer(mux),
+		ApiVersion: "v1",
```

Comment:
nit: we should stop using v1 even on tests 😅

Reply:
fixed